src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ByteSequenceComparer.cs

```diff
@@ -0,0 +1,42 @@
+﻿#nullable enable
+
+extern alias __uno;
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Uno.UI.SourceGenerators.XamlGenerator;
+
+internal static class ByteSequenceComparer
+{
+	// https://github.com/dotnet/roslyn/blob/6f47a0b611ab83ad1b7f8b404cc74098d37e283d/src/Compilers/Core/Portable/Collections/ByteSequenceComparer.cs#L24-L45
+	internal static bool Equals(ImmutableArray<byte> x, ImmutableArray<byte> y)
```

Comment:
After thinking about it, it should go into the Uno.Extensions folder as well.

Reply:
This one C:\Users\PC\Desktop\uno\src\Uno.Foundation\Uno.Core.Extensions\Uno.Core.Extensions.Equality ?

Reply:
Sure!